### PR TITLE
docs: update docs/docs/install-shells.mdx

### PR DESCRIPTION
**Pull request by UlanaXY**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

Reading the documentation, I was little confused about  what I should do with the `$PROFILE` variable. My first thought was that I should edit the variable itself, not the file lying under it. I think that this change may clarify this a little.

### CODE REVIEW

File: docs/docs/install-shells.mdx [hunk 1]
```diff
@@ -23,7 +23,11 @@ oh-my-posh --print-shell
 }>
 <TabItem value="powershell">
 
-Edit `$PROFILE` in your preferred PowerShell version and add the following line.
+Edit your PowerShell profile script, you can find its location under `$PROFILE` variable in your preferred PowerShell version. For example, using notepad:
```
Comment (JanDeDobbeleer) [inline docs/docs/install-shells.mdx]: can find its location under **the** `$PROFILE` variable

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]
One comment.

File: docs/docs/install-shells.mdx (hunk 1 above)
Comment (UlanaXY) [reply docs/docs/install-shells.mdx]: fixed

File: docs/docs/install-shells.mdx [hunk 2]
```diff
@@ -23,7 +23,11 @@ oh-my-posh --print-shell
 }>
 <TabItem value="powershell">
 
-Edit `$PROFILE` in your preferred PowerShell version and add the following line.
+Edit your PowerShell profile script, you can find its location under the `$PROFILE` variable in your preferred PowerShell version. For example, using notepad:
+```powershell
```
Comment (JanDeDobbeleer) [inline docs/docs/install-shells.mdx:28]: didn't see this but the linter will fail due to lack of a space between the paragraph and code block.

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: docs/docs/install-shells.mdx (hunk 2 above)
Comment (UlanaXY) [reply docs/docs/install-shells.mdx:28]: fixed

**Comment by JanDeDobbeleer** [comment]
@All-contributors please add @UlanaXY for doc

**Pull request merged**